modules/menu/menu.c

```diff
@@ -1012,6 +1013,39 @@ struct ua   *menu_ua_carg(struct re_printf *pf, const struct cmd_arg *carg,
 	return ua;
 }
 
+/**
+ * Decode url format
+ *
+ * @param w  Pl correspond the string is in URL format will be decoded
+ */
+void url_decode(struct pl* w)
+{
+	char *dst = (char*)w->p;
+	char *src = (char*)w->p;
+	int len = w->l;
```

Comment:
w->l is `size_t` so len should be `size_t` too.

Reply:
Thank you